package.json

```diff
@@ -63,8 +63,8 @@
   "dependencies": {
     "broccoli-funnel": "^2.0.1",
     "broccoli-merge-trees": "^2.0.0",
-    "browserslist": "^4.3.4",
     "chalk": "^2.3.0",
+    "ember-cli-babel": "^7.1.3",
```

Comment:
are we sure that this does not cause any unintended side-effects? 1. because `ember-cli-babel` might do something at app-buildtime and 2. because it's Babel 7, but we're still using Babel 6 to build the Ember.js assets

Reply:
My understanding is that `ember-cli-babel` will only apply to addon source code, so `ember-source`s addon code, which doesn't exist. There's definitely the discrepancy with Babel 6, but since that is a _pre_publish step and fundamentally disconnected from `ember-cli-babel`, I figured it would make more sense to use the latest Babel instead. We can change this though if it seems problematic.

Reply:
yeah, I just looked at https://unpkg.com/ember-source@3.5.1/lib/index.js, and this does indeed seem safe to do.